src/mongocrypt-crypto.c

```diff
@@ -188,7 +195,10 @@ static bool
 _crypto_aes_256_cbc_decrypt (_mongocrypt_crypto_t *crypto, aes_256_args_t args)
 {
    mongocrypt_status_t *status = args.status;
-   if (args.key->len != MONGOCRYPT_ENC_KEY_LEN) {
+
+   BSON_ASSERT_PARAM (crypto);
+
+   if (!args.key || args.key->len != MONGOCRYPT_ENC_KEY_LEN) {
```

Comment:
Suggest asserting `args.key` as not-null instead.

Reply:
:+1: Change coming in next commit.